counterparty-core/counterpartycore/lib/backend/__init__.py

```diff
@@ -0,0 +1,30 @@
+from counterpartycore.lib import backend, config, exceptions
+
+
+def list_unspent(source, allow_unconfirmed_inputs):
+    # first try with Bitcoin Core
```

Comment:
have you tested that this works?

Reply:
In unit test we are testing with a mock of electr and the the fallback is tested
In regtest we are testing for now only with bitcoin core 
and there is No UTXOs found with Bitcoin Core and Electr